CONTRIBUTING.md

```diff
@@ -130,41 +122,38 @@ Issue #190: Short summary of the issue
 ```
 
 Code
-====
+----
 
 This section include some tips that will help you to push source code.
 
-::: {.note}
-::: {.admonition-title}
-Note
-:::
-
-We recommend using [Python virtual
-environments](https://docs.python.org/3/tutorial/venv.html) to cleanly
-separate Qiskit from other applications and improve your experience.
-:::
-
-Setup with an environment
--------------------------
+**Note:**
+> We recommend using [Python virtual environments](https://docs.python.org/3/tutorial/venv.html) 
```

Comment:
To be consistent with Terra (which recommends Anaconda), it might be better to not recommend a specific environment. We can say something like
```We recommend using a self-contained environment, such as Python virtual environments or Anaconda environments, to cleanly separate Qiskit from other applications and improve your experience.```